src/styles/index.scss

```diff
@@ -1,3 +1,134 @@
+$mainAccent:#060b35;
+$secondary: #616070;
+$blueAccent: #00acdc;
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__content {
+    margin: 32px 16px 16px;
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__name {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainAccent;
+    margin: 40px 0 0;
+  }
+
+  &__productCode {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
` font-weigh` by default is 400 - redudant
```suggestion

```

Reply:
✅